source/views/components/filter/apply-filters.js

```diff
@@ -53,10 +54,16 @@ export function applyListFilter(filter: ListType, item: any): boolean {
 
 export function applyOrListFilter(
 	filterValue: ListItemSpecType[],
-	itemValue: string,
+	itemValue: string | string[],
 ): boolean {
 	// An item passes, if its value is in the filter's selected items array
-	return filterValue.map(f => f.title).includes(itemValue)
+	if (!Array.isArray(itemValue)) {
+		return filterValue.map(f => f.title).includes(itemValue)
+	}
+
+	const valueToCheckAgainst = filterValue.map(f => f.title.toString())
```

Comment:
I don't think we need toString here, do we?

Reply:
Flow gets mad without it, since title could be a number